src/pages/en/guides/deploy/aws.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Deploy your Astro Site to AWS
+description: How to deploy your Astro site to the web using AWS.
+layout: ~/layouts/DeployGuideLayout.astro
+---
+
+[AWS](https://cloud.google.com/) is a full-featured web app hosting platform that can be used to deploy an Astro site.
+
+## How to deploy
+
+Deploying your project to AWS requires using the [AWS console](https://aws.amazon.com/console/). (Most of these actions can also be done using the [AWS CLI](https://aws.amazon.com/cli/)). This guide will walk you through the steps to deploy your site to AWS from the most basic method then adding more services on top to improve cost efficiency and performance.
+
+### S3 Bucket
+
+S3 is the starting point of any application. It is where your proyect files and other assets are stored. S3 charges for file storage and number requests. You can find more information about S3 in the [AWS documentation](https://aws.amazon.com/s3/).
+
+1. Create an S3 bucket with your project's name.
+
+:::note
+The bucket name should be globally unique. We recommend combination of your project name and the domain name of your site.
+:::
```

Comment:
If you indent this, it will be parsed as part of step 1 and nested accordingly.

```suggestion
    :::note
    The bucket name should be globally unique. We recommend combination of your project name and the domain name of your site.
    :::
```